drivers/sensors/azoteq_iqs5xx.c

```diff
@@ -0,0 +1,247 @@
+// Copyright 2023 Dasky (@daskygit)
+// Copyright 2023 George Norton (@george-norton)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "azoteq_iqs5xx.h"
+#include "pointing_device_internal.h"
+#include "wait.h"
+
+#ifndef AZOTEQ_IQS5XX_ADDRESS
+#    define AZOTEQ_IQS5XX_ADDRESS 0x74 << 1
```

Comment:
Not a fan of the pre-shifted address, but both pre-shifted and shifted are used all over the place.

Reply:
Can't say I really have any preference either way. Maybe a PR for the future?

Reply:
Probably good practice to at least wrap it in brackets.

Reply:
Added brackets in 87fc5402ce3382124aeb9009fdc41f698ca4e62f thanks!